libraries/AP_GPS/AP_GPS.h

```diff
@@ -157,9 +174,8 @@ class AP_GPS
         return status(primary_instance);
     }
 
-    // Query the highest status this GPS supports
+    // Query the highest status this GPS supports (always reports GPS_OK_FIX_3D for the blended GPS)
```

Comment:
Let's escalate this to @tridge.  It doesn't seem useful to me and this PR is tough enough as it is.  If @tridge says let's add it then I'll go ahead and do it.